Source/sources/sponge_nd.F90

```diff
@@ -37,8 +37,8 @@ subroutine ca_sponge(lo,hi,state,state_lo,state_hi,source,src_lo,src_hi, &
     real(rt)         :: state(state_lo(1):state_hi(1),state_lo(2):state_hi(2),state_lo(3):state_hi(3),NVAR)
     real(rt)         :: source(src_lo(1):src_hi(1),src_lo(2):src_hi(2),src_lo(3):src_hi(3),NVAR)
     real(rt)         :: vol(vol_lo(1):vol_hi(1),vol_lo(2):vol_hi(2),vol_lo(3):vol_hi(3))
-    real(rt)         :: dx(3), dt, time
-    real(rt), value  :: mult_factor
+    real(rt)         :: dx(3)
+    real(rt), value  :: dt, time, mult_factor
```

Comment:
why are we making these `value`?  Shouldn't we be consistent with other routines?

Reply:
This is part of a general code transformation I am trying to make that will aid us for GPUs: when we are passing scalar, read-only values to the GPU, it makes the code a lot simpler if we pass them by value rather than by reference (if we do the latter, then we have to create an explicit allocation on the GPU for them).